public/app/core/services/backend_srv.ts

```diff
@@ -66,6 +66,11 @@ export class BackendSrv {
     var requestIsLocal = options.url.indexOf('/') === 0;
     var firstAttempt = options.retry === 0;
 
+    if (!options.url.match('https?://') && this.contextSrv && this.contextSrv.user && this.contextSrv.user.orgId) {
```

Comment:
I wasn't sure, the `requestIsLocal` check seemed to indicate that there might be non-local urls so I figured we shouldn't send the `X-Grafana-Org-Id` header to the external servers if that's the case.  As for the extra checks, that's just me being overly cautious so that if for some reason `contextSrv` isn't there or doesn't contain what we expect then we just don't send the header rather than causing a javascript error.